config/app.config

```diff
@@ -3,7 +3,7 @@
 //---------------------------------------------
 BuildMajor = 2022
 BuildMinor = 5
-BuildRev = 1
+BuildRev = 2
```

Comment:
Please update; we already have 2022.5.2 on data-int.

Reply:
ok